components/Buttons/src/MDCButton.m

```diff
@@ -503,7 +503,6 @@ - (void)animateButtonToHeightForState:(UIControlState)state {
 #pragma mark - BackgroundColor
 
 - (void)setBackgroundColor:(nullable UIColor *)backgroundColor {
-  // Turns out swift ignores NS_UNAVAILABLE
```

Comment:
After more consideration: I think we are LGTM by just putting a  comment explaining whats going on. This is mostly to make it clear that if the button is not in a normal state it wont be calling super.backgroundcolor = backgroundColor with the passed in value.